README.md

```diff
@@ -27,8 +37,290 @@ pip install -e .
+**A**perture **M**asking **I**nterferometry (AMI) data from major existing
+facilities:
+[NIRISS](https://jwst-docs.stsci.edu/near-infrared-imager-and-slitless-spectrograph)
+on the JWST (first scientific interferometer operating in space),
+[SPHERE](https://www.eso.org/sci/facilities/paranal/instruments/sphere.html) and
+[VISIR](https://www.eso.org/sci/facilities/paranal/instruments/visir.html) from
+the European Very Large Telescope (VLT) and
+[VAMPIRES](https://www.naoj.org/Projects/SCEXAO/scexaoWEB/030openuse.web/040vampires.web/indexm.html)
+from SUBARU telescope (and more to come).
+
+We focused our efforts to propose a user-friendly interface, though different
+sub-classes allowing to (1) **Clean** the reduced datacube from the standard
+instrument pipelines, (2) **Extract** the interferometrical quantities
+(visibilities and closure phases) using a Fourier sampling approach and (3)
+**Calibrate** those quantities to remove the instrumental biases.
+
+Looking for a quickstart into AMICAL? Just have a look to the example scripts
+made for [NIRISS](example_NIRISS.py) and [SPHERE](example_NIRISS.py).
+
+In addition (4), we include two external packages called
+[CANDID](https://github.com/amerand/CANDID) and
+[Pymask](https://github.com/AnthonyCheetham/pymask) to **analyse** the final
+outputs obtained from a binary-like sources (star-star or star-planet). We
+interfaced these stand-alone packages with AMICAL to quickly estimate our
+scientific results (e.g.: separation, position angle, contrast ratio, contrast
+limits, etc.) using different approaches (chi2 grid, MCMC, see
+[example_analysis.py](example_analysis.py) for details).
+
+## Tutorial
+
+In this tutorial, we will go through the different possibilities of AMICAL. You
+can find a detailed description of the principal functions and associated
+parameters in their docstrings (easily accessible with good editors, e.g.:
+vscode).
+
+- [Step 1: clean and select data](#step-1-clean-and-select-data)
+- [Step 2: extract observables](#step-2-extract-observables)
+- [Step 3: calibrate V2 & CP](#step-3-calibrate-v2--cp)
+- [Step 4: analyse with CANDID and
+  Pymask](#step-4-analyse-with-candid-and-pymask)
+
+To begin, we just need to import AMICAL library:
+
+```python
+import amical
+```
+
+### Step 1: clean and select data
+
+The major part of data coming from general pipelines (applying dark, flat,
+distorsion, etc.) are not enought compliant with the Fourier extracting method
+developed within AMICAL.
+
+The first step of AMICAL consists to clean the data in different way:
+
+- Remove residual sky background (`sky`=True, `r1`, `dr`)
+- Crop and center the image (`isz`, `f_kernel`=3),
+- Apply windowing (`apod`, `window`),
+- Apply bad pixels correction (`bad_map`=None, `add_bad`=[],
+  `remove_bad`=False).
+
+We use a 2d gaussian interpolation to replace the bad pixels (same as
+[here](https://docs.astropy.org/en/stable/convolution/)). The input `bad_map` is
```

Comment:
honest question: input to what ?
(plus, you probably mean "argument", rather than "input")